src/timeunit.ts

```diff
@@ -92,5 +103,22 @@ export function format(timeUnit: TimeUnit, abbreviated = false): string {
     out.push(timeComponents.join(':'));
   }
 
-  return out.length > 0 ? out.join(' ') : undefined;
+  if (out.length > 0) {
+  // clean up quarter formatting remainders
+   const template = '{{datum.data | time:\'' + out.join(' ') + '\'}}';
+   return template.replace(new RegExp('{{datum.data \\| time:\'\'}}', 'g'), '');
+  } else {
+   return undefined;
+  }
+}
+
+/** Returns true if container contains the containee, false otherwise. */
+export function containsTimeUnit(container: TimeUnit, containee: TimeUnit) {
```

Comment:
should we rename `container` to `fullTimeUnit` and `containee` => `singleTimeUnit`  ?